### ISSUE TITLE
KHR_vulkan_glsl typo in an example.

### ISSUE THREAD

**Issue by NicolBolas** [KhronosGroup/Vulkan-Docs#466]
The specification contains this example:

````
layout(...) uniform sampler s;   // handle to filtering information
layout(...) uniform texture2D t; // handle to a texture
in vec2 tCoord;
...
texture2D(sampler2D(t, s), tCoord);
````

However, `texture2D` is not a function; it is a type. It *used* to be a function, but KHR_vulkan_glsl explicitly removes that function:

>The following features are removed:
> * ...
> * the already deprecated texturing functions (e.g., texture2D())

So that should probably be `texture(sampler2D(t, s), tCoord);`

**Comment by ParticlePeter** [comment]
Please consider adding this example to the spec. From the current spec (v1.0.42) we get to know that it is possible to combine a separate sampler and image within a shader (section [13.1.3 Sampled Image](https://www.khronos.org/registry/vulkan/specs/1.0/html/vkspec.html#descriptorsets-types)), but not how it is actually done. I had a hard time figuring this out and found it 'accidentally' in the KHR_vulkan_glsl description, no other source is available afaik.

**Comment by oddhack** [comment]
This should be fixed in the 1.0.49 spec update.
